index.html

```diff
@@ -1,259 +1,250 @@
+          </ul>
+          <button class="btn2" type="button">
+              See this project
+          </button>
+      </div>
+    </div>
+
+    <div class="six projectcard">
+        <div class="thumbnail">
+        </div>
+        <div class="project-summary">
+          <h2>Project's name goes here</h2>
+          <ul class="languagebadge">
+              <li class="language">HTML/CSS</li>
+              <li class="language">Ruby on Rails</li>
+              <li class="language">>Javascript</li>
+          </ul>
+          <button class="btn2" type="button">
+              See this project
+          </button>
+      </div>
+    </div>
+
+    <div id="projectimg1"></div>
+    <div id="projectimg2"></div>
+    <div id="projectimg3"></div>
+    <div id="projectimg4"></div>
+  </section>
+  <section class="about" id="about">
+    <div class="container-about">
+      <h2 class="aboutheader">About</h2>
+      <p class="introabout">
+        Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut,
+      </p>
+      <button type=button class="btn3">Get my CV</button>
+    </div>
+    <div class="container-about2">
+      <ul class="grid-container">
+        <li class="aboutcard">
+          <ul class="grid-item1">
+            <li class="about-title">
+              <h3>Languages</h3>
+            </li>
+            <li>JavaScript (es5/es6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item2">
+            <li class="about-title">
+              <h3>Frameworks</h3>
+            </li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item3">
+            <li class="about-title">
+              <h3>Skills</h3>
+            </li>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>Github</li>
+            <li>GitLab</li>
+            <li>terminal</li>
+          </ul>
+        </li>
+        <li class="aboutcard">
+          <ul class="grid-item4">
+            <li><img src="/css/dots.png" width="170"  alt="dots"></li> 
+          </ul>
+      </ul>
+    </div>
+  </section>
+  <form action="https://formspree.io/f/mayaljan" method="post" id="form">
+    <fieldset class="contact">
+      <h2 class="contactheader">Get Started</h2>
+      <p class="introcontact">Lorem ipsum dolor sit amet,
+        consectetur adipiscing elit,
+        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        Ut enim ad minim veniam,
+        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. .</p>
+      <div class="form-group">

```

Comment:
- [ ] Kindly include the `background-image` used in the form section of Figma's template for your design. 

## Your Form section

![image](https://user-images.githubusercontent.com/77106962/144726120-fd012820-1e37-4c48-afe7-adbfc8e06efa.png)

## Figma's Form Section (Template 5)

![image](https://user-images.githubusercontent.com/77106962/144726134-67667579-3dd9-4118-99a9-244cfdeb4d16.png)